consoles/VNC.pm

```diff
@@ -921,6 +936,65 @@ sub _receive_zrle_encoding ($self, $x, $y, $w, $h) {
     return $res;
 }
 
+# wrapper to make testing easier
+sub _read_socket ($socket, $data, $data_len, $offset) {
+    return read($socket, $data, $data_len, $offset);    # uncoverable statement
+}
```

Comment:
As I see the message about missing coverage I guess you could further simplify that to

```suggestion
sub _read_socket ($socket, $data, $data_len, $offset) { read($socket, $data, $data_len, $offset) }  # uncoverable statement
```

though you might still need to add another "uncoverable statement" or multiple to make Devel::Cover happy. Or mock the internal "read" and call `_read_socket` in tests anyway

Reply:
Are you planning to look into https://github.com/os-autoinst/os-autoinst/pull/2417#issuecomment-1863126651 still?

Reply:
There's now 100 % patch coverage.